pkg/service/flipt/transport/service.go

```diff
@@ -117,12 +121,23 @@ func (s *Service) instance() (offlipt.Client, error) {
 	var err error
 
 	s.once.Do(func() {
+		u, uerr := url.Parse(s.address)
+		if uerr != nil {
+			err = fmt.Errorf("connecting %w", uerr)
+		}
+
+		if u.Scheme == "https" || u.Scheme == "http" {
+			s.client = sdk.New(sdkhttp.NewTransport(s.address)).Flipt()
+
+			return
+		}
+
 		conn, cerr := s.connect()
 		if cerr != nil {
 			err = fmt.Errorf("connecting %w", cerr)
 		}
-		fliptSdk := sdk.New(sdkgrpc.NewTransport(conn))
-		s.client = fliptSdk.Flipt()
+
+		s.client = sdk.New(sdkgrpc.NewTransport(conn)).Flipt()
```

Comment:
It would be good if we could support the SDK options somewhere here.
Mostly so we can do authentication: https://pkg.go.dev/go.flipt.io/flipt/sdk/go#Option